Index: untitled/src/Figura2/Bus.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package Figura2.untitled.src.Figura2;\r\n\r\npublic class Bus {\r\n    private String patente;\r\n    private String marca;\r\n    private String modelo;\r\n    private int nroAsientos;\r\n\r\n    public Bus(String patente, String marca, String modelo, int nroAsientos) {\r\n        this.patente= patente;\r\n        this.marca= marca;\r\n        this.modelo=modelo;\r\n        this.nroAsientos= nroAsientos;\r\n    }\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/untitled/src/Figura2/Bus.java b/untitled/src/Figura2/Bus.java
--- a/untitled/src/Figura2/Bus.java	(revision 2a499de16a998be58c49df6d91da9b8b312e9486)
+++ b/untitled/src/Figura2/Bus.java	(date 1777170213998)
@@ -1,7 +1,7 @@
 package Figura2.untitled.src.Figura2;
 
 public class Bus {
-    private String patente;
+    private String patente;.
     private String marca;
     private String modelo;
     private int nroAsientos;
Index: untitled/src/Figura2/Pasajero.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/untitled/src/Figura2/Pasajero.java b/untitled/src/Figura2/Pasajero.java
new file mode 100644
--- /dev/null	(date 1777088461263)
+++ b/untitled/src/Figura2/Pasajero.java	(date 1777088461263)
@@ -0,0 +1,26 @@
+package Figura2;
+
+public class Pasajero extends Persona {
+    private Nombre nomContacto;
+    private String fonoContacto;
+
+    public Pasajero (idPersona idPersona, Nombre nombreCompleto, String telefono, Nombre nomContacto, String fonoContacto){
+        super (idPersona, nombreCompleto);
+        setTelefono (telefono);
+        this.nomContacto = nomContacto();
+        this.fonoContacto = fonoContacto;
+    }
+        public Nombre getNomContacto(){
+        return nomContacto;
+    }
+    public void setNomContacto(Nombre nom){
+        this.nomContacto=nom;
+    }
+    public String getFonoContacto(){
+        return fonoContacto;
+    }
+
+    public void setFonoContacto(String fonoContacto) {
+        this.fonoContacto = fonoContacto;
+    }
+}
\ No newline at end of file
